src/node_util.cc

```diff
@@ -289,34 +291,66 @@ static void GuessHandleType(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(fd, 0);
 
   uv_handle_type t = uv_guess_handle(fd);
-  const char* type = nullptr;
+  uint32_t type{0};
```

Comment:
I think we can use a enum here and then create the array in the binding. That removes the hard-coding here and in the JS land.

Reply:
Would you mind, if we merge it at this state, and follow up on a different PR?

Reply:
I am fine with a TODO

Reply:
I added the TODO. Thank you for your patience.